### DISCUSSION TITLE
Any benefit to enabling iBoost model if not using predbat to control it?

### DISCUSSION THREAD

**Discussion by mudge** [springfall2008/batpred#1258] [Q&A] [answered]
Is there any benefit to enabling [predbat's iBoost model with `switch.predbat_iboost_enable`](https://springfall2008.github.io/batpred/customisation/#iboost-model-options) if you're not actually respecting predbat's resulting decisions about when to enable it (by having some sort of device to physically press the iBoost's "boost" button)?

That is to say, does it improve predbat's planning at all to factor in the presence of an iBoost or does it not meaningfully change its charge/discharge behaviour?

I have my iBoost set with an export threshold of 250W so my GivEnergy Giv-Bat 9.5 Gen 2 takes priority when there is excess PV energy.

**Comment by gcoan** [comment, marked as answer]
If you generally are not going to respect what Predbat suggests and you don't have an automation such as a fingerbot to push the button, then I suggest you don't turn iboost on.

If you turn it on then predbat assumes the iboost will be operated per the plan and it will optimise the battery level, charging, etc on the assumption that the excess solar is used.
Subsequently if you don't press the button then Predbat will just recalculate at the next 5 minute run anyway.  It's just the long term trend and forecast export revenue will be wrong as it will keep thinking the iboost is using the excess energy.

With iboost turned off then predbat will have a more accurate long term prediction.  Of course if you do at some point manually turn on iboost then Predbat will see the changed export figure and recalculate the plan anyway.

There's probably no major impact whether you turn iboost enable on or off, but if you tend not to follow what it suggests I would lean to not turning it on so the plan is more accurate as a result.
